### DISCUSSION TITLE
Access to your account has been suspended

### DISCUSSION THREAD

**Discussion by Miguel1995c** [community/community#139418] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Buenas, disculpen.

He creado esta cuenta para crear un ticket y recuperar mi cuenta principal y aun no recibo una solución, el ticket que he enviado
aun no tiene respuesta.
También he creado otro ticket con la cuenta principal afectada obviamente entrando al siguiente link.
https://github.com/contact
la opcion:: ¿no puedes iniciar sesion? relleno el formulario:::: Cuenta bloqueada: Mi cuenta ha sido bloqueada o suspendida:::::
y sigo con los pasos correspondiente:

deberían ampliar las opciones para que el usuario pueda recuperar su cuenta sin la necesidad de realizar un ticket para soporte técnico, ya que por lo visto esta muy saturado y la solución de parte de ellos tardan en llegar.

En la documentacion de Github te dan 3 horas para que puedas resetear la clave al momento que te salga cuenta suspendida.
si te pasas de las tres horas ya pasa a ser problema de soporte tecnico y tienes que crear un ticket para que te den una solución al problema.

saludos

**Comment by ebndev** [comment]
Hi @Miguel1995c,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
